phpseclib/File/X509.php

```diff
@@ -3117,12 +3118,13 @@ public function removeExtension($id)
      *
      * @param string $id
      * @param array $cert optional
+     * @param string $path
      * @access public
      * @return mixed
      */
-    public function getExtension($id, $cert = null)
+    public function getExtension($id, $cert = null, $path)
```

Comment:
I think that should be `$path = null`. As is not all the calls to `getExtension` have a third parameter:

```php
$issuerAltName = $this->getExtension('id-ce-subjectAltName', $issuer->currentCert);
```

Also, that's how it's defined with `getExtensionHelper`:

```php
    private function getExtensionHelper($id, $cert = null, $path = null)
```